fsw/src/to_lab_app.c

```diff
@@ -249,64 +232,60 @@ int32 TO_LAB_EnableOutput(const TO_LAB_EnableOutput_t *data)
 {
     const TO_LAB_EnableOutput_Payload_t *pCmd = &data->Payload;
 
-    (void) CFE_SB_MessageStringGet(TO_LAB_Global.tlm_dest_IP, pCmd->dest_IP, "",
-                                   sizeof (TO_LAB_Global.tlm_dest_IP),
-                                   sizeof (pCmd->dest_IP));
+    (void)CFE_SB_MessageStringGet(TO_LAB_Global.tlm_dest_IP, pCmd->dest_IP, "", sizeof(TO_LAB_Global.tlm_dest_IP),
+                                  sizeof(pCmd->dest_IP));
```

Comment:
It seems like it should have caught this indent

Reply:
Looks right to me?

Reply:
The tool is just splitting the args differently between the 2 calls.  This is where a human probably would have done it the same on both lines, so it visually looks the same between the two calls to the same function.  
Not a big issue, but also not ideal.

Reply:
Yep, it was rendering strangely on my browser.